gui/src/main.rs

```diff
@@ -60,18 +66,24 @@ impl Default for GuiConfig {
 }
 
 pub struct Gui {
-    pub api: PostieApi,
-    pub config: Arc<RwLock<GuiConfig>>,
```

Comment:
I converted the single `config` property that held numerous other properties to be individual properties. Since the ui was constantly accessing `config` to check different fields for rendering having the shared property just made things harder to work with.

Now that everything is its own pointer to memory it is easier to access each field